build.gradle

```
@@ -0,0 +1,78 @@
+import java.nio.file.Files
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING
+
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    // Core dependency for Logstash Output Google Cloud Storage
+    compile "com.google.cloud:google-cloud-storage:1.39.0"
+
+    // Dependencies of com.google.cloud:google-cloud-storage:1.39.0
+    compile "com.fasterxml.jackson.core:jackson-core:2.9.2"
```

```suggestion
    compileOnly "com.fasterxml.jackson.core:jackson-core:2.9.2"
```
since jackson is already loaded by logstash-core we should make it a compileOnly dependency here.